ui/server.tsx

```diff
@@ -7,10 +7,13 @@ import { App } from 'ui/App'
 import { Config, ConfigProvider } from 'ui/components/ConfigProvider'
 import { Document } from 'ui/Document'
 import { initApollo } from 'ui/lib/initApollo'
-
 export default async function(req: Request, res: Response, config: Config) {
   let html = ''
-
+  const assetPaths = '../public/assets.json'
+  const routes = (await import(assetPaths)).default as { client: string }
```

Comment:
I'm pretty sure using `import` here will cause server ssr to fail. It'll use the cached version of the module rather than re-loading the file on disk.

Reply:
There is a reason I was using string-based loading.

Reply:
Ugh, I think I was confused about which file this is...